ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -434,6 +434,7 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
     env:
       DISCONNECTED: "true"
+      GOVERSION: "1.21"
       IP_STACK: v6
     workflow: assisted-baremetal-operator-ztp
```

Comment:
What about overriding at workflow level? At least all our jobs would run with the same go version, which could ease investigations.

Reply:
Good idea! Modified it to the workflows instead - what do you think?

Reply:
Great! Thanks!